doc/source/cluster/commands.rst

```diff
@@ -0,0 +1,168 @@
+Cluster Launcher Commands
+=========================
+
+This document overviews common commands for using the Ray Cluster Launcher.
+See the :ref:`Cluster Configuration <cluster-config>` docs on how to customize the configuration file.
+
+Launching a cluster (``ray up``)
+--------------------------------
+
+This will start up the machines in the cloud, install your dependencies and run any setup commands that you have, configure the Ray cluster automatically, and prepare you to scale your distributed system. See :ref:`the documentation <ray-up-doc>` for ``ray up``.
+
+.. tip:: The worker nodes will start only after the head node has finished starting. To monitor the progress of the cluster setup, you can run `ray monitor <cluster yaml>`.
+
+.. code-block:: shell
+
+    # Replace '<your_backend>' with one of: 'aws', 'gcp', 'kubernetes', or 'local'.
+    $ BACKEND=<your_backend>
+
+    # Create or update the cluster.
```

Comment:
Small nit, could you possibly define what "`your_backend`" is (i.e. `gcp/aws/azure/local`, etc.)

Reply:
yep, it's inline in the comment.

Reply:
(above the line)